includes/class-wc-tracker.php

```diff
@@ -113,7 +113,7 @@ private static function is_jetpack_staging_site() {
 	 *
 	 * @return array
 	 */
-	private static function get_tracking_data() {
+	public static function get_tracking_data() {
```

Comment:
While it's great that you have added unit tests to your changes, turning a private method to public for the sake of making it testable is not a good idea imho; anything public adds a maintenance burden to the code since we need to keep backwards compatibility further on.

As an alternative, you can invoke `send_tracking_data` in your test instead, using the `pre_http_request` filter to capture the arguments and prevent the request from being actually sent. I'll add more details in another comment in the test file.

For other cases where it isn't possible to bypass the behavior of build-int/WordPress/WooCommerce functions by using hooks it's worth taking a look at [the testing infrastructure in place](https://github.com/woocommerce/woocommerce/blob/master/src/README.md#interacting-with-legacy-code).